src/crocoddyl_reaching.cpp

```diff
@@ -71,11 +71,9 @@ namespace panda_torque_mpc
         // Collision constraints
         auto runningConstraintModelManager =  boost::make_shared<crocoddyl::ConstraintModelManager>(state);
         auto terminalConstraintModelManager =  boost::make_shared<crocoddyl::ConstraintModelManager>(state);
-        Eigen::VectorXd lower_bound(1);
-        Eigen::VectorXd upper_bound(1);
-
-        lower_bound << 5e-2;
-        upper_bound << std::numeric_limits<double>::infinity();
+        Eigen::VectorXd lb_col(1), ub_col(1);
+        lb_col << config.collision_safety_margin;
+        ub_col << std::numeric_limits<double>::infinity();
```

Comment:
```suggestion
        Eigen::VectorXd col_lower_bound(1);
        Eigen::VectorXd col_upper_bound(1);
        col_lower_bound << config.collision_safety_margin;
        col_upper_bound << std::numeric_limits<double>::infinity();
```
Personally, I like this combination more. `lb` and `up` are a bit ambiguous IMO